amundsen_application/static/js/components/TableDetail/EditableSection/tests/index.spec.tsx

```diff
@@ -65,5 +66,18 @@ describe("EditableSection", () => {
       const { wrapper } = setup(null, child);
       expect(wrapper.childAt(1).text()).toBe(child);
     });
+
+    it("renders button when readOnly=false", () => {
+      expect(wrapper.find(".btn").length).toEqual(1);
+    });
+
+    it("renders does not add button when readOnly=true", () => {
+      const { wrapper } = setup({readOnly: true}, <TagInput/>);
+      expect(wrapper.find(".btn").length).toEqual(0);
```

Comment:
```suggestion
      expect(wrapper.find(".edit-button").length).toEqual(0);
```
All buttons in this application have the class `.btn`, so I'm suggesting the className specifically for the edit button in case there is ever another button.

Reply:
ok makes sense!